FWCore/ParameterSet/python/Config.py

```diff
@@ -533,11 +533,17 @@ def extend(self,other,items=()):
                 if not item.hasLabel_() :
                     item.setLabel(name)
             elif isinstance(item,Schedule):
-                self.__setattr__(name,item)
```

Comment:
I think this is a general issue with `Process.load(...)`: for all other objects it makes sense to replace the exiting ones, but for the `schedule` I think it makes more sense to append it.